### PR TITLE
Audio submits

### PR DESCRIPTION
**Pull request by clementzach**
This PR will let sycamore's submits_and_deliveries.csv output file have numbers for submissions of audio surveys and text surveys. 

Some context: Much of the code in this PR was added in [sycamore_tableau_function](https://github.com/onnela-lab/forest/pull/119). I had someone else review my code for that function, but they didn't have write access. 

If it would lead to less work, I could merge [sycamore_tableau_function](https://github.com/onnela-lab/forest/pull/119) before starting the review process for this PR.

### CODE REVIEW

**Review by hackdna — CHANGES_REQUESTED** [review]
I've merged #119 but now this PR has conflicts with `develop`.

File: forest/sycamore/submits.py [hunk 1]
```diff
@@ -128,9 +129,36 @@ def generate_survey_times(
...
+            example, this would come from read_json(config_path)["surveys"][0]
+        audio_survey_id_dict: Output from get_audio_survey_id_dict. Dict with
+            survey prompts as keys and survey IDs as values
+    Returns:
+        List of all question IDs for an individual. For audio surveys, it will
+        create a "question ID" that is identical to the survey ID
+
+    """
+    question_ids = []
+    for q in survey_dict["content"]:
```
Comment (hackdna) [inline forest/sycamore/submits.py:146]: `q` -> `question`?

File: forest/sycamore/tests/test_functions.py [hunk 2]
```diff
@@ -8,16 +8,20 @@
 from forest.sycamore.common import (aggregate_surveys,
                                     aggregate_surveys_no_config,
                                     aggregate_surveys_config,
-                                    filename_to_timestamp,
                                     read_user_answers_stream,
                                     read_aggregate_answers_stream,
                                     get_choices_with_sep_values)
-from forest.sycamore.submits import (gen_survey_schedule,
+from forest.sycamore.utils import filename_to_timestamp, read_json
+from forest.sycamore.submits import (gen_survey_schedule, get_question_ids,
```
Comment (hackdna) [inline forest/sycamore/tests/test_functions.py:15]: utils after submits

File: forest/sycamore/tests/test_functions.py [hunk 3]
```diff
@@ -8,16 +8,20 @@
...
                                      get_all_interventions_dict,
                                      survey_submits, summarize_submits,
                                      survey_submits_no_config)
 from forest.sycamore.responses import (agg_changed_answers_summary,
                                        format_responses_by_submission)
+from forest.sycamore.read_audio import (get_audio_survey_id_dict,
```
Comment (hackdna) [inline forest/sycamore/tests/test_functions.py:21]: read_audio before responses

File: forest/sycamore/utils.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import re
+
+import pandas as pd
+
+
+def read_json(study_dir: str) -> dict:
+    """Read a json file into a dictionary
+
+    Args:
+        study_dir (str):  filepath to json file.
```
Comment (hackdna) [inline forest/sycamore/utils.py:18]: This is not actually a dir but a json file path, so the name should reflect that (e.g., `filepath`).

File: forest/sycamore/utils.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    Returns:
+        A dict representation of the json file
+    """
+    with open(study_dir, "r") as f:
+        dictionary = json.load(f)
```
Comment (hackdna) [inline forest/sycamore/utils.py:23]: Simply: `return json.load(f)`?

**Review by hackdna — CHANGES_REQUESTED** [review]

File: forest/sycamore/utils.py (hunk 5 above)
Comment (hackdna) [reply forest/sycamore/utils.py]: You still need to open the file with a context manager to make sure the file is closed when processing is finished:
```python
with open(study_dir, "r") as f:
    return json.load(f)
```

**Review by hackdna — CHANGES_REQUESTED** [review]
Just one outstanding item I think.

**Comment by clementzach** [comment]
Awesome, thank you so much!

**Review by hackdna — APPROVED** [review]

**Pull request merged**